spec/factories/curator/filestreams/audios.rb

```diff
@@ -1,9 +1,11 @@
+# frozen_string_literal: true
+
 FactoryBot.define do
   factory :curator_filestreams_audio, class: 'Curator::Filestreams::Audio' do
     association :file_set_of, factory: :curator_digital_object
     ark_id { "commonwealth:#{SecureRandom.hex(5)}" }
     file_set_type { 'Curator::Filestreams::Audio' }
-    file_name_base { Faker::File.file_name(dir: 'foo') }
+    file_name_base { Faker::File.file_name(ext: 'wav') }
```

Comment:
`file_name_base` values should not have extensions (e.g. `06_01_008447`, not `06_01_008447.tif`)